src/google-maps/google-map/google-map.ts

```diff
@@ -372,14 +372,14 @@ export class GoogleMap implements OnChanges, OnInit, OnDestroy {
    */
   getCenter(): google.maps.LatLng {
     this._assertInitialized();
-    return this.googleMap.getCenter();
+    return this.googleMap.getCenter()!;
```

Comment:
Why are we asserting here but elsewhere we are changing the return type to accept undefined?

Reply:
I probably missed it when I was reverting the non-null assertions. Changed.